Modix.Services/Core/UserService.cs

```diff
@@ -97,13 +97,12 @@ public async Task<IUser> GetUserAsync(ulong userId)
                 : await (await DiscordClient.GetGuildAsync(AuthorizationService.CurrentGuildId.Value))
                     .GetUserAsync(userId);
 
-            if (user == null)
-                throw new InvalidOperationException($"Discord user {userId} does not exist");
-
             if (user is IGuildUser guildUser)
                 await TrackUserAsync(guildUser);
 
-            return user;
+            return user
```

Comment:
I would prefer not to change the implementation of GetUserAsync here. Instead, see my comment on `AutocompleteController.cs`.

Reply:
Why?

Reply:
Oh, I didn't realize we had refactored out usage of it entirely. In that case, I'd rather remove it later and continue to use `GetUserInformationAsync`, as that's what's used by the `!info` module.

Reply:
Should I remove it now then? Seeing as nothing else uses it, I might as well, right?

Reply:
Sure. If it breaks something, we'll find out in CI :D

Reply:
10/10 I like that strategy